Templates/DefaultGem/template.json

```
@@ -1,360 +1,256 @@
 {
     "template_name": "DefaultGem",
+    "template_restricted_platform_relative_path": "Templates/DefaultGem",
     "restricted_name": "o3de",
-    "restricted_platform_relative_path": "Templates",
-    "origin": "The primary repo for DefaultGem goes here: i.e. http://www.mydomain.com",
-    "license": "What license DefaultGem uses goes here: i.e. https://opensource.org/licenses/MIT",
-    "display_name": "DefaultGem",
-    "summary": "A short description of DefaultGem.",
-    "canonical_tags": [],
+    "restricted_platform_relative_path": "Templates/DefaultGem",
+    "origin": "https://github.com/o3de/o3de",
+        "license": "https://opensource.org/licenses/Apache-2.0 Or https://opensource.org/licenses/MIT",
+    "display_name": "Default Gem Template",
```

Whitespacing is off with the "license" line
```suggestion
    "origin": "https://github.com/o3de/o3de",
    "license": "https://opensource.org/licenses/Apache-2.0 Or https://opensource.org/licenses/MIT",
    "display_name": "Default Gem Template",
```